docs/modules/data-structures/pages/cpmap.adoc

```diff
@@ -0,0 +1,47 @@
+= CPMap
+[[cpmap]]
+
+Hazelcast `CPMap` is a distributed implementation of a minimal key-value interface. It supports 
+atomic `compareAndSet`, `delete`, `get`, `put`, `remove` and `set` operations. Since `CPMap` is a 
+distributed key-value implementation, these operations involve remote calls and thus their 
+performance differs from non-distributed key-value data structures, e.g. `java.util.Hashmap`. 
+`CPMap` is an enterprise only feature.
+
+WARNING: There is no unsafe variant of `CPMap`, unlike other CP data structures. Therefore, CP must
+be enabled before making use of `CPMap`.
+
+The following example creates an instance of a `CPMap` in the `default` CP group, then updates its
+contents via the operations listed previously.
```

Comment:
Latin needs to be avoided in docs (I know, it is about in various places through the docs, but not all users have English as a first language and no one has Latin as a language; I'm on a mission to eradicate it ;-)) - so, just the outstanding comments removing Latin (and the comment rewording for "should", which implies it isn't that important, or it might or might not be the case, or it is just an advisory)

Thanks!

Reply:
Done.